README.md

```diff
@@ -22,73 +22,43 @@ Check out the version of this repo you wish to use (typically main), and then ru
 This will ensure the right packages, users, groups is configured, and set up
 jobrunner and other services as needed.
 
+## Base assumptions
 
-## Development
-
-Directory layout:
-
-* ./scripts
-  * various bash scripts to perform specific actions, designed to be idempotent
-* ./etc
-  * system configuration files/templates
-* ./tpp-backend
-  * scripts and config for tpp backend
-* ./emis-backend
-  * scripts and config for emis backend
-* ./test-backend
-  * scripts and config for our test backend
-* ./emis-access
-  * scripts for managing emis-access VM
-* ./jobrunner
-  * scripts and config templates for jobrunner
-* ./osrelease
-  * scripts and config templates for osrelease
-* ./keys/$USER
-  * public keys to add to ssh for $USER
-* ./developers
-  * developers gh account list. To be replaced by authenticated API call to
-    job-server later, perhaps.
-* ./tests
-  * test scripts, run in docker container by makefile
-* ./run-in-lxd.sh
-  * runner for above tests. Uses LXD to provide a VM-like environment to run scripts on.
-
-## Testing
-
-To run tests, we use LXD to provide and isolate VM-like environment. You will
-need to have LXD installed and configured, and `shiftfs` enabled. Currently,
-this probably only works on Ubuntu.
+ * Ubuntu server (20.04 baseline)
+ * Internet access to {jobs,docker-proxy,github-proxy}.opensafely.org
+ * Internet access to an official ubuntu archive
+ * SSH access for developers to the backend host
+ * sudo access on the host in some form
+ * Just bash and git needed on the host to bootstrap backend.
 
-https://linuxcontainers.org/lxd/getting-started-cli/
+## Components
 
-Quickstart for ubuntu:
+### Workspace state
 
-```
-snap install lxd --classic
-sudo snap set lxd shiftfs.enable=true
-sudo lxd init --auto
-```
+Currently in `/srv/high_privacy` and `/srv/medium_privacy`. Files owned by
+`jobrunner` user. Medium privacy files can be read by `reviewers` group.
 
-To run tests:
+### job-runner
+ 
+Repo: https://github.com/opensafely-core/job-runner
 
-    make test
+Manages the jobs and their state. Currently deployed in `/srv/jobrunner` as a git
+checkout and managed by a systemd unit.  Plan is to move this to docker
+soonish.
 
-To run specific tests (tab completes)
+### osrelease
 
-    make tests/$TEST
+Repo: https://github.com/opensafely-core/output-publisher
 
-If you specify DEBUG=1, then you will be dropped into a shell inside the docker
-container after the tests has run.
+Command line tool to release files. Currently deployed via `sudo pip install`,
+which is problematic, so will be switching something else soonish.
```

Comment:
sorry if this is a silly question but why is this problematic?

Reply:
Not silly at all, but not somthing I wanted to put in the readme :) 

Using `sudo pip install` has caused some issues when updating the files, as the ownership reverts to root only, which means no one else can run it.  There's a few ways to fix this, but we also need to add dependencies, so I wan't to figure out a better way. Ideally, this would be part of the osrelease docker image, with a simple wrapper script to run it.